dockerfiles/Dockerfile.openvino-csharp

```diff
@@ -37,9 +37,9 @@ RUN apt update && \
     cd /etc/apt/sources.list.d && \
     echo "deb https://apt.repos.intel.com/openvino/2021 all main">intel-openvino-2021.list && \ 
     apt update && \
-    apt -y install intel-openvino-dev-ubuntu18-2021.1.110 && \
+    apt -y install intel-openvino-dev-ubuntu18-2021.2.200 && \
     cd ${INTEL_OPENVINO_DIR}/install_dependencies && ./install_openvino_dependencies.sh && \
-    cd ${INTEL_OPENVINO_DIR} && rm -rf documentation && cd deployment_tools/ && rm -rf model_optimizer open_model_zoo demo && cd inference_engine && rm -rf samples && \
+    cd ${INTEL_OPENVINO_DIR} && rm -rf documentation data_processing && cd deployment_tools/ && rm -rf model_optimizer tools open_model_zoo demo && cd inference_engine && rm -rf samples  && \
```

Comment:
consider to use runtime package to avoid removing unnecessary content:  intel-openvino-runtime-ubuntu18-2021.2.200